src/api/handlers/guild.ts

```diff
@@ -462,20 +462,40 @@ export async function getMembers(
     throw new Error(Errors.MISSING_INTENT_GUILD_MEMBERS);
   }
 
-  const result = (await RequestManager.get(
-    endpoints.GUILD_MEMBERS(guildID),
-    options,
-  )) as MemberCreatePayload[];
+  const members = new Collection<string, Member>();
+  let loops = 1;
+  while (options?.limit ?? 1 > members.size) {
+    console.log(
+      `Paginating get members from REST. #${loops} / ${
+        Math.ceil((options?.limit ?? 1) / 1000)
```

Comment:
if limit is undefined what would this be?

Reply:
If limit is undefined it would be 1

Reply:
Also, can you remove the console.log statements?

Reply:
Why? Skillz wanted an console.log here